File: Labs/Lab2/Anagram.py
def isAnagram(s,t):
    if len(s) != len(t):
        return "s and t are not anagrams"
    elif sorted(s) == sorted(t):
        return "s and t are anagrams"
    else:
        return "s and t are not anagrams"

s = "eleven plus twd"
t = "twelve plus one"
print(isAnagram(s.strip(),t.strip()))

## If length of string is not equal it cannot be an anagram
## Returns anagram if the sorted string s and t are the same, other it returns false

File: Labs/Lab2/Anagram.java
import java.util.Arrays;

public class Anagram {
    public static void main(String[] args) {
        String s = "eleven plus two";
        String t = "twelve plus one";
        System.out.println(isAnagram(s.trim(), t.trim()));
    }

    public static String isAnagram(String s, String t) {
        if (s.length() != t.length()) {
            return "s and t are not anagrams";
        } else {
            char[] sArray = s.toCharArray();
            char[] tArray = t.toCharArray();
            Arrays.sort(sArray);
            Arrays.sort(tArray);
            if (Arrays.equals(sArray, tArray)) {
                return "s and t are anagrams";
            } else {
                return "s and t are not anagrams";
            }
        }
    }
}
